### DISCUSSION TITLE
Always getting "subtitles from amara.org community"  when transcripting in hungsrian with a simple curl call

### DISCUSSION THREAD

**Discussion by matehun00** [openai/whisper#1536] [Q&A]
I have a lua script that does the simple curl call, and it works fine when I set the language to english "en" but when I set to hungarian "hu", It always gives me the same output "Feliratok az Amara.org közösségétől" which basically means "subtitles from amara.org community"

**Comment by glangford** [comment]
This is a well known problem, see

* https://github.com/openai/whisper/discussions/928

**Reply by matehun00** [reply]
The difference that I'm having that it always gives me that output and not just sometimes

**Reply by glangford** [reply]
The training data used for Hungarian might be even more biased than other languages. Just fyi there is also this discussion specific to hu:
* https://github.com/openai/whisper/discussions/710

**Comment by erde01-web** [comment]
> I have a lua script that does the simple curl call, and it works fine when I set the language to english "en" but when I set to hungarian "hu", It always gives me the same output "Feliratok az Amara.org közösségétől" which basically means "subtitles from amara.org community"

Ja, da war ein Auge. Da ist ein Hund. Sorry ich mach grad Fotos vom Gesichtern oder Tieren in Asche.

**Reply by erde01-web** [reply]
Kann man hier auf Fotos oder Videos posten?
